galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -12,6 +12,31 @@ servers:
         default: "8080"
 
 paths:
+  /ras/requestors:
+    get:
+      summary: Get all Requestors
+      parameters:
+        - name: requestors
+          in: query
+          description: defines ascending or descending
+          required: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Array of Requestors
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
It is bad practive to have the top level json object as an array,  there are security implication and can lead to bufferoverflows etc.
You will need to define an component to describe {"requestors":["xxx"]} type structure.